.github/workflows/cover.yaml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches:
+    - master
```

Comment:
this would run only post submit on the master branch right? Is there a way to run it on PRs as informative, just to show how much coverage the PR is adding/decreasing?

Reply:
I think that's what we want?

Reply:
hmm, not sure how that ends up working in the ui...you might have to manually setup the compare between master and the PR branch 🙁 I'm admittedly new to configuring codecov and mostly have enjoyed using it for other projects, let me poke around

Reply:
maybe we just use this as visualization on the master branch for now, since https://github.com/kubernetes/test-infra/pull/18873/files is going to calculate coverage on PRs.

Reply:
do you know what kind of visualization we get from prow? I swear there was a link somewhere but I can't seem to find it. If we get good data from that, I'm happy to close this all together. I was excited to see @cpanato's PR

Reply:
I don't think we get any visuals from prow, that's the problem... @cpanato please correct me if I'm wrong.

Reply:
Peep coverage at the very bottom https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-coverage-unit/1295352716532387841

doesn't seem interactive // have good breakout coverage on the files, but it's there-ish

Reply:
also https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit&graph-metrics=coverage&sort-by-failures=

(graph -> coverage, and sort-by-failures)

not incredible, but not bad either. I personally still prefer the codecov overall ui